sdk/servicebus/azure-messaging-servicebus/src/test/java/com/azure/messaging/servicebus/ServiceBusReceiverAsyncClientIntegrationTest.java

```diff
@@ -22,178 +24,165 @@
+
         final String messageId = UUID.randomUUID().toString();
-        final ServiceBusMessage message = TestUtils.getServiceBusMessage(CONTENTS, messageId, 0);
+        final ServiceBusMessage message = TestUtils.getServiceBusMessage(CONTENTS_BYTES, messageId, 0);
+
+        sendMessage(message).block();
 
         // Assert & Act
-        StepVerifier.create(sender.send(message).then(receiver.peek()))
-            .assertNext(receivedMessage -> assertTrue(receivedMessage.getProperties().containsKey(MESSAGE_TRACKING_ID)))
+        StepVerifier.create(receiver.peek())
+            .assertNext(receivedMessage -> assertMessageEquals(receivedMessage, CONTENTS_BYTES, messageId, 0))
             .verifyComplete();
     }
 
     /**
      * Verifies that we can schedule and receive a message.
      */
-    @Test
-    void sendScheduledMessageAndReceive() {
+    @MethodSource("receiverTypesProvider")
+    @ParameterizedTest
+    void sendScheduledMessageAndReceive(MessagingEntityType entityType) {
         // Arrange
+        setSenderAndReceiver(entityType);
+
         final String messageId = UUID.randomUUID().toString();
-        final String contents = "Some-contents";
-        final ServiceBusMessage message = TestUtils.getServiceBusMessage(contents, messageId, 0);
+        final ServiceBusMessage message = TestUtils.getServiceBusMessage(CONTENTS_BYTES, messageId, 0);
         final Instant scheduledEnqueueTime = Instant.now().plusSeconds(2);
-
-        sender.scheduleMessage(message, scheduledEnqueueTime)
-            .delaySubscription(Duration.ofSeconds(3))
-            .block();
-
-        // Assert & Act
-        StepVerifier.create(receiver.receive().take(1))
-            .assertNext(receivedMessage -> {
-                assertArrayEquals(contents.getBytes(), receivedMessage.getBody());
-                assertTrue(receivedMessage.getProperties().containsKey(MESSAGE_TRACKING_ID));
-            })
-            .verifyComplete();
-    }
-
-    /**
-     * Verifies that we can schedule and receive multiple messages.
-     */
-    @Test
-    void sendMultipleScheduledMessageAndReceive() {
-        // Arrange
-        final String messageId1 = UUID.randomUUID().toString();
-        final String messageId2 = UUID.randomUUID().toString();
-        final ServiceBusMessage message1 = TestUtils.getServiceBusMessage(CONTENTS, messageId1, 0);
-        final ServiceBusMessage message2 = TestUtils.getServiceBusMessage(CONTENTS, messageId2, 0);
-        final Duration duration = Duration.ofSeconds(10);
-        final Instant scheduledEnqueueTime = Instant.now().plus(duration);
         final ReceiveAsyncOptions options = new ReceiveAsyncOptions().setEnableAutoComplete(false);
 
-        sender.scheduleMessage(message1, scheduledEnqueueTime)
-            .block(TIMEOUT);
-        sender.scheduleMessage(message2, scheduledEnqueueTime)
-            .block(TIMEOUT);
+        sender.scheduleMessage(message, scheduledEnqueueTime).block();
 
```

Comment:
Why we are not using TIMEOUT ?

Reply:
In IntegrationTestBase, we set the `StepVerifier.setDefaultTimeout()`, so it'll use that. Practically, we can remove all the .block(TIMEOUT).